src/library/process.js

```diff
@@ -70,10 +87,16 @@ export const processMessage = async (AWS, region, service, account,
   }, msgHandler);
   console.log(`processMessage: INFO: Result from worker is ${JSON.stringify(procRes, null, 4)}`);
 
+  // CR: Mickey: is the assumption below that the serviceUserData + account found in the previous
+  //    block (ln43-62) may now be out of date?  
+  //    I would also lean towards making this a named helper function to make it more
+  //    clear what its purpose is like "saveServiceDataToAccount"
   if (itemExists(procRes, 'serviceUserData')) {
     if (typeof procRes.serviceUserData !== 'object') {
       throw new Error('Service specific user data should be an object');
     }
+    // CR: Mickey: if we're going to stick with direct DB access, ths exact 15 lines also
+    //    exists 40 lines above.  Time for a "getAccount" helper?
```

Comment:
@MickeyMcc Although I like the graphQL approach, I believe direct DB approach will be better at least for this framework as we control the framework to ensure that no unwanted DB access occurs and also because directDB access is definitely faster than introducing another layer in between. Definitely, it is time for a getAccount helper.